docs/source/orquesta/start.rst

```diff
@@ -9,10 +9,10 @@ Like any |st2| action, an Orquesta workflow requires an action metadata file in
 is the path relative to ``/opt/stackstorm/packs/<mypack>/actions`` for the workflow definition.
 
 Let's start with a very basic Orquesta workflow named ``examples.orquesta-sequential`` for the
-``examples`` pack. The workflow definition for this example is provide below. This workflow is a
-very simple example that run a few echoes, piece together the output, and return a response. A task
-can reference any registered |st2| action directly. In this example, the tasks call ``core.echo``.
-an action in |st2| that just echo back the message like the shell echo command. The ``core.echo``
+``examples`` pack. The workflow definition for this example is provided below. This workflow is a
+very simple example that runs a few echoes, pieces together the output, and returns a response. A task
+can reference any registered |st2| action directly. In this example, the tasks calls ``core.echo``,
```

Comment:
*the tasks* is 3rd person plural and should be conjugated as _call_.

Reply:
Yep. There are lots of small errata on the page.